packages/core/src/base-chart.ts

```diff
@@ -166,6 +166,7 @@ export class BaseChart {
 				// Perform the draw or update chart
 				if (initialDraw) {
 					this.initialDraw();
+					this.drawTitle(this.options.chartTitle);
```

Comment:
drawTitle itself has access to `this.options.chartTitle`, no reason to pass it in